typst-package/manual.typ

```diff
@@ -1,435 +1,358 @@
 #import "./lib.typ": *
+#import "./lib.typ"
+#import "@preview/tidy:0.2.0"
+#import "@preview/tablex:0.0.8": tablex
+#import "./tidy_style.typ"
+
+// TODO: replace tidy_style with tidy.styles.default next version
+#set page(
```

Comment:
we can add a white background in page so texts are still visible when github is in dark theme